jonathankamau/news-headlines
# #152709875 Add webpack and react app configurations

#### What does this PR do?
The app currently is not able to run locally and does not display the html view.
This ensures that the webpack configurations are set which are necessary for the app to run.

#### Description of Task to be completed?
create the file `webpack.config.js` and add the necessary configurations. create the file `.babelrc` and add the presets. Add the `Html Webpack Plugin` to the list of dependencies.

#### How should this be manually tested?
clone the repo locally, ensure all the required dependencies are installed then run the app by running the command `yarn start`

#### What are the relevant pivotal tracker stories?
[#152709875](https://www.pivotaltracker.com/story/show/152709875)
